metricbeat/module/mysql/_meta/config.yml

```
@@ -1,7 +1,9 @@
 - module: mysql
-  #metricsets:
-  #  - status
+  metricsets:
+    - status
```

Why doing this change? Metricsets use to be intentionally commented out, so default ones are enabled if no one is specified. Here we are enabling status in the provided configuration, but both status and performance are default metricsets.
```suggestion
  #metricsets:
  #  - status
```